uiautomationutilities/perf_tool/PerfDataRecord.cs

```diff
@@ -0,0 +1,75 @@
+﻿//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+
+using System;
+
+namespace PerfClTool.Measurement
+{
+    internal class PerfDataRecord
+    {
+        public static readonly string ValueNotApplicable = "NA";
+        public PerfDataRecord(string timeStamp, string marker, string time, string thread, string cpuUsed, string cpuTotal, string residentSize, string virtualSize, string wifiSent, string wifiRecv, string wwanSent, string wwanRecv, string appSent, string appRecv, string battery, string systemDiskRead, string systemDiskWrite)
```

Comment:
Can we clean up the formatting to prevent having to side-scroll past 100-ish chars? This, and in all files